### PR TITLE
Change github markdown styling

### PR DESCRIPTION
**Pull request by dannycobussen**
Change styling of the github markdown for our documentation.
- Added more width to body and remove border
- Add print stylesheet

### CODE REVIEW

File: resources/github-markdown.css [hunk 1]
```diff
@@ -26,10 +26,14 @@ THE SOFTWARE.
...
+  01-12-2015
+    Changed width from 45 em to 720px, removed border &
+    added media query for print;
+ */
+
 body {
-  width: 45em;
-  border: 1px solid #ddd;
-  outline: 1300px solid #fff;
+  width: 720px;
```
Comment (flurin) [inline resources/github-markdown.css]: Can you keep this in EM sizing?

Comment (dannycobussen) [inline resources/github-markdown.css]: Done :)

**Comment by flurin** [comment]
Mhhh not ok now

**Comment by dannycobussen** [comment]
As discussed, only added the print media query for now.

File: resources/github-markdown.css [hunk 2]
```diff
@@ -26,6 +26,11 @@ THE SOFTWARE.
 
 */
 
+/*
+  01-12-2015
+    added media query for print;
```
Comment (flurin) [inline resources/github-markdown.css:31]: Why this comment? Dont' we have Git for that?

Comment (dannycobussen) [inline resources/github-markdown.css:31]: Purpose of this was that it's easier to see that it's not the original github markdown, but has some modifications.

**Pull request merged**